lib/source-info.coffee

```diff
@@ -31,12 +32,32 @@ module.exports =
     currentLine: ->
       @_currentLine ||= unless @_currentLine
         editor = atom.workspace.getActiveTextEditor()
-        cursor = editor and editor.getCursor()
+        cursor = editor and editor.getLastCursor()
         if cursor
           cursor.getBufferRow() + 1
         else
           null
 
+    minitestRegExp: (text)->
+      @_minitestRegExp ||= unless @_minitestRegExp 
+        value = text.match(/"(.*?)"/) if text?
+        if value
+          value[1]
+        else
+          ""
+
+    isMiniTest: ->
+      i = @currentLine() - 1
+      regExp = new RegExp(/^(\s+)(should|test|it)\s+['""'](.*)['""']\s+do\s*(?:#.*)?$/)
```

Comment:
Continuing on from the previous comment, this will match both a minitest spec test and an rspec test. That is bad. Instead, you could check the first line of the file, which usually contains `require 'spec_helper'` for rspec.